db/projects/quarkchain/project.yaml

```diff
@@ -0,0 +1,87 @@
+name: QuarkChain
+web:
+  - 'https://quarkchain.io/'
+description: 'QuarkChain is a high-capacity peer-to-peer transactional system.'
+start_date: '2018-06-02'
+assets:
+  - id: qkc
+    name: QuarkChain
+    type: token
+    ledger: ethereum
+    symbol: QKC
+    contract_address: '0xea26c4ac16d4a5a106820bc8aee85fd0b7b2b664'
+    max_supply: 10000000000
+    decimals: 18
+    token_standard: ERC-20
+    start_date: '2018-06-02'
+    webids:
+      coinmarketcap: quarkchain
+      cryptocompare: qkc
+      cryptocompare/id: '899553'
+    ico_properties:
+      start_date: '2017-11-25'
+      end_date: '2018-01-28'
+      total_cap: 10000000000
+      raised: 20000000 USD
+    first_distribution:
+      - title: 'Public participants'
+        percent: 20
+      - title: 'Mining & community'
+        percent: 45
+      - title: 'Team'
+        percent: 15
+      - title: 'Advisors'
+        percent: 5
+      - title: 'Fund'
+        percent: 15
```

Comment:
It's nothing like mistake, but just for your information. For this field you can choose either percent: number (represent the percent of the token as you used) or amount: number (represent the total number of the tokens).

Reply:
ok, I found percentages easier to dig up